Resources/Computing/ARCComputingElement.py

```diff
@@ -290,10 +290,14 @@ def killJob(self, jobIDList):
       jobList = [jobIDList]
 
     gLogger.debug("Killing jobs %s" % jobIDList)
+    jobs = []
     for jobID in jobList:
-      job = self.__getARCJob(jobID)
-      if not job.Cancel():
-        gLogger.debug("Failed to kill job %s. CE(?) not reachable?" % jobID)
+      jobs.append(self.__getARCJob(jobID))
+
+    # JobSupervisor is able to aggregate jobs to perform bulk operations and thus minimizes the communication overhead
+    job_supervisor = arc.JobSupervisor(self.usercfg, jobs)
+    if not job_supervisor.Cancel():
+      gLogger.debug("Failed to kill at least one of the jobs. CE(?) not reachable?")
```

Comment:
`warn` or `error` instead of `debug`?

Reply:
I replaced it by `return S_ERROR` like in the other Computing Elements.